server/api/src/main/scala/com/prisma/api/mutations/SqlMutactions.scala

```diff
@@ -121,6 +127,18 @@ case class SqlMutactions(dataResolver: DataResolver) {
     x.flatten.toVector
   }
 
+  def getMutactionsForScalarLists2(where: NodeSelector, args: CoolArgs) = {
```

Comment:
do we still need the old `getMutactionsForScalarLists `?

Reply:
Yes, now one delivers Mutactions and the other DBActions. I already cleaned up the naming in the follow up. And maybe can unite them again once we have the control flow abstraction. But for now they do different things.